### ISSUE TITLE
Linux is not supported

### ISSUE THREAD

**Issue by walidkorchi** [TechStudent10/DiscordRPC#23]
This is more a feature request rather than an issue to support Unix operating systems. I've been expecting this mod to work on Arch but nothing comes up on my Discord profile activity... well until I saw in the README markdown file that this mod only works for Windows.

Please consider supporting unix systems.

**Comment by TechStudent10** [comment]
I'm a Linux user myself and I've been using [wine-discord-ipc-bridge](https://github.com/0e4ef622/wine-discord-ipc-bridge) to do so.

I've been intending to add this into the mod itself, however it is pretty low priority.

I'm gonna keep the issue for if/when I come back to to this

**Comment by walidkorchi** [comment]
> I'm a Linux user myself and I've been using [wine-discord-ipc-bridge](https://github.com/0e4ef622/wine-discord-ipc-bridge) to do so.
> 
> I've been intending to add this into the mod itself, however it is pretty low priority.
> 
> I'm gonna keep the issue for if/when I come back to to this

Can you at least show the ropes of how [wine-discord-ipc-bridge](https://github.com/0e4ef622/wine-discord-ipc-bridge) works for Linux users by mentioning it in the README file, like as a workaround? I gave it a try but failed at setting it up with Steam right before the game launch in settings options

**Comment by JizzyOnGithub** [comment]
yeah same how did you setup the ipc thing i cant get it to work

**Comment by TechStudent10** [comment]
Open up Steam

Navigate to Geometry Dash

Click Settings, then Properties 

In the properties panel, find Launch Options.

Go to the wine-discord-ipc-bridge repository and make sure you have both the shell script in the repo downloaded, as well as the .EXE file in the release. These should be in the same directory. 

After that, copy the direct path to the wine-discord-ipc-bridge shell script. Put that right before everything else in the launch options. 

Open GD and test.

**Comment by walidkorchi** [comment]
@TechStudent10 It does not seem to work:

https://streamable.com/09luw1

**Comment by TechStudent10** [comment]
> @TechStudent10 It does not seem to work:
> 
> https://streamable.com/09luw1

the WINEDLLOVERRIDES part should be at the very beginning

**Comment by walidkorchi** [comment]
> the WINEDLLOVERRIDES part should be at the very beginning

doesn't work either, here's the full prompt: `WINEDLLOVERRIDES="xinput9_1_0=n,b" /home/studio/wine-discord-ipc-bridge/winediscordipcbridge-steam.sh %command%`

**Comment by TechStudent10** [comment]
Did you make the file executable by using chmod +x?

**Comment by walidkorchi** [comment]
> Did you make the file executable by using chmod +x?

Sure did: https://streamable.com/4afz1c

**Comment by TechStudent10** [comment]
> > Did you make the file executable by using chmod +x?
> 
> Sure did: https://streamable.com/4afz1c

Ah I see. After you open the game, open your extended profile and click on Activity. If you see your presence working, then I think we've found the problem.

**Comment by walidkorchi** [comment]
I also have this when cleaning up registered games on Discord and actually renaming `geometrydash` to `Geometry Dash`, which works but it yeah the mod doesn't seem to send the RPC info to Discord. Do you have an idea?

![image](https://github.com/TechStudent10/DiscordRPC/assets/87608619/f4791f6a-7bc1-49f0-a4a3-0ee9ec7ad3c8)

**Comment by TechStudent10** [comment]
Try removing Geometry Dash from registered games in Discord

**Comment by walidkorchi** [comment]
@TechStudent10 Already tried.

**Comment by TechStudent10** [comment]
At this point I really don't know. I don't even use Steam to run GD anymore but instead use Wine. If you could find a way to run a program within Steam/GD's Wine Prefix, then you'd be able to run the EXE from wine-discord-ipc-bridge before opening GD, and the RPC would work. Other than that, I don't know what the issue could be.

**Comment by Cuteistfox** [comment]
BTW support for this with arRPC is blocked by OpenAsar/arrpc#83 (also you should colapse the comments of this issue)(and also @Walidoux you can [use steam tinker launch to do the rpc for you](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Discord-IPC-Bridge-(WDIB))
